pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -659,6 +659,40 @@ class Build {
         }
     }
 
+    def waitForANodeToBecomeActive(def label) {
+        def NodeHelper = context.library(identifier: 'openjdk-jenkins-helper@master').NodeHelper
+
+        if (NodeHelper.nodeIsOnline(label)) {
+            return
+        }
+
+        context.println("No active node matches this label: " + label)
+
+        int activeNodeTimeout = 0
+        if (${buildConfig.ACTIVE_NODE_TIMEOUT}.isInteger()) {
+            activeNodeTimeout = ${buildConfig.ACTIVE_NODE_TIMEOUT} as Integer
+        }
+
+
+        if (activeNodeTimeout > 0) {
+        context.println("Will check again periodically until either one comes online, or " + ${buildConfig.ACTIVE_NODE_TIMEOUT} + " minutes (ACTIVE_NODE_TIMEOUT) has passed.")
+            int x = 0
+            while (x < activeNodeTimeout) {
+                sleep(60 * 1000)
```

Comment:
You could use groovy's advanced sleep for better brevity. See https://github.com/AdoptOpenJDK/openjdk-build/pull/2108

Reply:
Sorted, though I don't see how this makes it more brief. Thoughts?

Reply:
Mainly for brevity sake. Its a lot easier when debugging to see `time: activeNodeTimeout, unit: "MINUTES"` rather than a multiplication sum in the code